app/controllers/WKRemoteDataStoreController.scala

```diff
@@ -259,6 +259,17 @@ class WKRemoteDataStoreController @Inject()(
       }
     }
 
+  def getDataset(name: String, key: String, datasetId: ObjectId): Action[AnyContent] =
```

Comment:
```suggestion
  def getDataSource(name: String, key: String, datasetId: ObjectId): Action[AnyContent] =
```

(as it does not return a Dataset object, which is what is stored in postgres, I’d rather follow the DataSource terminology in this context)